lib/rhc/commands/app.rb

```diff
@@ -425,7 +425,8 @@ def show(app_name)
         case options.gears
         when 'quota'
           opts = {:as => :gear, :split_cells_on => /\s*\t/, :header => ['Gear', 'Cartridges', 'Used', 'Limit'], :align => [nil, nil, :right, :right]}
-          table_from_gears('echo "$(du --block-size=1 -s 2>/dev/null | cut -f 1)"', groups, opts) do |gear, data, group|
+          # quota 'blocks' are actually kb, so have to multiply by 1024, see 'man quota'
+          table_from_gears('echo "$(quota | egrep -v \'quotas|Filesystem\' | awk \'{ print $2 * 1024 }\')"', groups, opts) do |gear, data, group|
```

Comment:
Mystery solved with the 'echo "$(quota...)" ', thank you!
I'm looking at fedora 23 man quota.  Weird.  It's in the 'DESCRIPTION' section.
I'm fine with keeping it '1024' or I could read it from mount.h, either way.
:)